consensus/enclave/impl/build.rs

```diff
@@ -25,4 +26,41 @@ fn main() {
 
     fs::write(&env.out_dir().join("target_features.rs"), &contents)
         .expect("Could not write target feature array");
+
+    rerun_if_env_changed!("FEE_SPEND_PUBLIC_KEY");
+    rerun_if_env_changed!("FEE_VIEW_PUBLIC_KEY");
+
+    let mut fee_spend_public_key = [0u8; 32];
+    let mut fee_view_public_key = [0u8; 32];
+
+    let default_fee_spend_pub = "26b507c63124a2f5e940b4fb89e4b2bb0a2078ed0c8e551ad59268b9646ec241";
```

Comment:
Should document how this was generated, and its connection to the test code private keys.